### PR TITLE
#184797863  Admin get all orders

### PR DESCRIPTION
**Pull request by GasanaJr**
#### What does this PR do?
This PR adds an admin route to get all orders.
#### Description of Task to be completed?
- creation of `admin route` to view all orders
#### How should this be manually tested?
- clone this repo `git clone https://github.com/atlp-rwanda/e-comm-team-amigos-bn`
- switch your working directory `cd e-comm-team-amigos-bn`
- create an `.env` and update it with `.env-example`
- install all dependencies by running `npm install`
- in your terminal run `npm start`
#### Test the endpoints using Postman:
- get all orders by admin`/product/admin`
#### Any background context you want to provide?
- PostgreSQL installed in your local machine
- Node JS installed in your local machine
#### Screenshots
- Screenshot 2023-04-01 at 3 00 07 AM
- Screenshot 2023-04-01 at 3 10 33 AM
#### What are the relevant pivotal tracker stories?
Story ID: [#184797863](https://www.pivotaltracker.com/n/projects/2632829/stories/184797863)

### CODE REVIEW

**Review by wilbrord2 — APPROVED** [review]

**Review by nshutijonathan — APPROVED** [review]
LGTM

**Review by AbdulKhaliq59 — APPROVED** [review]

**Review by AbdulKhaliq59 — APPROVED** [review]
resolve conflict unless It LGTM

**Comment by wilbrord2** [comment]
Can you please remove this conflicts

File: src/controllers/order.controller.js [hunk 1]
```diff
@@ -218,3 +218,19 @@ exports.updateOrderStatus = async (req, res, next) => {
...
+export const adminGetOrders = async (req, res) => {
+    try {
+        const orders = await models.Order.findAll()
+        if (orders) {
+            return res.status(200).json({
+                message: "All orders in the Database",
+                orders: orders
+            });
+        }
+    }
```
Comment (Habinezajanvier) [inline src/controllers/order.controller.js]: Check also if no order found and send a 404 status code with error message

**Review by Habinezajanvier — CHANGES_REQUESTED** [review]

File: src/controllers/order.controller.js (hunk 1 above)
Comment (GasanaJr) [reply src/controllers/order.controller.js]: It is done now

**Pull request merged**